courses/free-courses-en.md

```diff
@@ -463,6 +469,7 @@
 * [Node and Express Tutorial](https://www.youtube.com/watch?v=TNV0_7QRDwY) - John Smilga, Coding Addict (YouTube)
 * [Node.js API Development for Beginners](https://www.udemy.com/course/node-js-api-tutorial/) (Udemy)
 * [Node.js Basics](https://www.youtube.com/playlist?list=PLqq-6Pq4lTTa-d0iZg41U2RDqECol9C5B) - Java Brains (Youtube)
+* [Node.js Tutorial for Beginners](https://www.youtube.com/watch?v=TlB_eWDSMt4) - Programming with Mosh (Youtube)
```

Comment:
:information_source: Not a playlist but 1h20m video with timeline in player bar

:warning: Some issues with creator attribution completion and right resource title

```markdown
* [Node.js Tutorial for Beginners: Learn Node in 1 Hour](https://www.youtube.com/watch?v=TlB_eWDSMt4) - Moshfegh Hamedani, Programming with Mosh (YouTube)
```